src/chrome/content/rules/Roblox-whitelist.xml

```diff
@@ -0,0 +1,8 @@
+<ruleset name="Roblox (whitelisted domains)">
```

Comment:
@zoracon Is this another whitelist way?

Reply:
These are not whitelisted, maybe they used to be. All of them actually work.